cfpb/consumerfinance.gov
# Related metadata: Remove category sidebar

According to @wpears 

> that module supported an initial version of the enforcement work which has now been replaced. It can safely be removed.

Originally added in https://github.com/cfpb/consumerfinance.gov/pull/5344/files

## Removals

- Category block option from related metadata block.

## How to test this PR

1. Pull branch, run migrations, edit a browse filterable page and view the related metadata sidefoot. Topics should be an option, but Categories should no longer be.

<img width="1434" alt="Screen Shot 2021-08-03 at 10 00 33 AM" src="https://user-images.githubusercontent.com/704760/128028807-579c13b4-892c-4949-bf85-90e3aa02b0d6.png">
